test-server/test-echo.c

```diff
@@ -269,9 +275,12 @@ int main(int argc, char **argv)
 			rate_us = atoi(optarg) * 1000;
 			break;
 #endif
+#ifdef WIN32
```

Comment:
The variable debug_level seems to be dead code under Windows. It is only defined (see first added #ifdef) and assigned with a command line argument (see second added #ifdef) but never actually used when being compiled with WIN32.